### PR TITLE
Add test cases for wego-install

### PR DESCRIPTION
**Pull request by rokshana-b**
Add test cases for `wego install`, as well as a couple of negative test cases for `wego` flags.

### CODE REVIEW

File: test/acceptance/test/wego_flux_tests.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	var err error
+
+	BeforeEach(func() {
+
+		By("Given I have a wego binary installed on my local machine", func() {
+			Expect(FileExists(WEGO_BIN_PATH)).To(BeTrue())
+		})
+	})
+
+	AfterEach(func() {
```
Comment (josecordaz) [inline test/acceptance/test/wego_flux_tests.go:26]: Wondering if this empty function has a particular usage 🙂

File: test/acceptance/test/wego_install_tests.go [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+	BeforeEach(func() {
+		WEGO_BIN_PATH = "/Users/rokshana/weaveworks/weave-gitops/bin/wego"
+
+		By("Given I have a wego binary installed on my local machine", func() {
+			Expect(FileExists(WEGO_BIN_PATH)).To(BeTrue())
+		})
+	})
+
+	AfterEach(func() {
```
[the remark this answers is not in the document]
Comment (rokshana-b) [reply test/acceptance/test/wego_install_tests.go:28]: This function performs any action that needs to be repeated after each test run. For now, we don't have any actions with such requirements. I could probably get rid of it...

[the remark this answers is not in the document]
Comment (josecordaz) [reply test/acceptance/test/wego_install_tests.go:28]: nice 🙂

**Review by iahmad9 — APPROVED** [review]
Refactored the tests.
